src/Examine/LuceneEngine/Search/LuceneQuery.cs

```diff
@@ -59,7 +59,7 @@ public IBooleanOperation GroupedNot(IEnumerable<string> fields, params IExamineV
 
         public string Category => _search.Category;
 
-        public IBooleanOperation NativeQuery(string query) => _search.NativeQuery(query);
+        public IBooleanOperation NativeQuery(string query, ISet<string> loadedFieldNames = null) => _search.NativeQuery(query, loadedFieldNames);
```

Comment:
optional parameters are binary breaking changes. These will need to be overloads.